value_test.go

```diff
@@ -85,17 +86,22 @@ func TestValueBasic(t *testing.T) {
 		{
 			Key:    []byte("samplekey"),
 			Value:  []byte(val1),
-			meta:   bitValuePointer,
+			meta:   0,
```

Comment:
This should not be zero. The value is bigger than value threshold so this pointer should not be zero.